src/js/generateRandomString.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+const generateRandomString = () => Math.random().toString(32).slice(2);
```

Comment:
maybe this one: 🙂 
```js
import {randomBytes} from 'crypto';

export const getRandomString= (length = 16, encoding='hex') => randomBytes(length).toString(encoding);
```

Reply:
We need a simple one. No need other imports.

Reply:
`crypto` is part of nodeJS itself so why not to use it :) - but this is just a suggestion

Reply:
@mpacholec but we use it in browser ?:P

Reply:
@aju https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto